products/sle15/profiles/anssi_bp28_minmal.profile

```diff
@@ -0,0 +1,122 @@
+documentation_complete: true
+
+metadata:
+    SMEs:
+        - abergmann
+
+title: 'ANSSI-BP-028 (minimal)'
+
+description: |-
+    This profile contains configurations that align to ANSSI-BP-028 v1.2 at the minimal hardening level.
+
+    ANSSI is the French National Information Security Agency, and stands for Agence nationale de la sécurité des systèmes d'information.
+    ANSSI-BP-028 is a configuration recommendation for GNU/Linux systems.
+
+    A copy of the ANSSI-BP-028 can be found at the ANSSI website:
+    https://www.ssi.gouv.fr/administration/guide/recommandations-de-securite-relatives-a-un-systeme-gnulinux/
+
+    Only the components strictly necessary to the service provided by the system should be installed.
+    Those whose presence can not be justified should be disabled, removed or deleted.
+    Performing a minimal install is a good starting point, but doesn't provide any assurance
+    over any package installed later.
+    Manual review is required to assess if the installed services are minimal.
+
+selections:
```

Comment:
@teacup-on-rockingchair Hi, you can also leverage the ANSSI control files in `controls/anssi.yml`.

You can define the profile selections with ` - anssi:all:minimal`
Similar to what we do in RHEL8:
https://github.com/ComplianceAsCode/content/blob/master/products/rhel8/profiles/anssi_bp28_minimal.profile#L19

This will include in the profile all rules in the Recommendations at Minimal level if the `prodtype` allows.

Reply:
Ahh, disregard my comment, just read the PR description, :)